massa-node/base_config/config.toml

```diff
@@ -367,17 +367,17 @@
     # [optional] port on which to listen for incoming bootstrap requests. You may need to change this to "0.0.0.0:port" if IPv6 is disabled system-wide.
     bind = "[::]:31245"
     # timeout to establish a bootstrap connection
-    connect_timeout = 15000
+    connect_timeout = 300000 # 5 mins
```

Comment:
Just so I'm clear on this, why did we have to increase this? Is this timeout checkout for each message received? If so, 5 mins seem a bit high, no? (someone could idle for almost 5 mins and then send 1 message and idle again..)

Reply:
Yeah that's just a leftover from previous tests, I hit it sometimes when the server was really busy which is why I left it, but I guess it's OK to restore it back

Reply:
Ok understood, IMO we basically should approach this as "if the server is too busy and we hit this timeout, then either we have to accept less clients, or we should calibrate it better with the test-framework."